addons/mail/static/src/components/composer_suggested_partners/composer_suggested_partners.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.chatter.ComposerSuggestedPartners" owl="1">
+        <!-- List of followers -->
+        <div class="o_composer_suggested_partners">
+            <t t-foreach='suggestedPartners' t-as='recipient'>
```

Comment:
fix for your issue: `suggestedPartners` is currently undefined in component. Need to start from an existing getter, in this current case `composer.thread.suggestedPartners`